great_expectations/data_context/data_context/abstract_data_context.py

```diff
@@ -980,23 +1007,32 @@ def get_validator_using_batch_list(
         Returns:
 
         """
-        if len(batch_list) == 0:
-            raise ge_exceptions.InvalidBatchRequestError(
-                """Validator could not be created because BatchRequest returned an empty batch_list.
-                Please check your parameters and try again."""
-            )
+        self._check_len_batch_list(batch_list)
+        execution_engine: ExecutionEngine = self.datasources[  # type: ignore[union-attr]
+            batch_list[-1].batch_definition.datasource_name
+        ].execution_engine
+        return self._get_validator_using_batch_list(
+            expectation_suite=expectation_suite,
+            batch_list=batch_list,
+            execution_engine=execution_engine,
+            include_rendered_content=include_rendered_content,
+            **kwargs,
+        )
 
+    # noinspection PyUnusedLocal
+    def _get_validator_using_batch_list(
```

Comment:
This is the old `get_validator_using_batch_list` without the part that retrieved the `execution_engine`.